sideloader/postinstall.sh

```diff
@@ -0,0 +1,9 @@
+pip="${VENV}/bin/pip"
+
+cd "${INSTALLDIR}/${REPO}/"
+
+$pip install -e "${INSTALLDIR}/${REPO}/"
+
+springboard="${VENV}/bin/springboard"
+
+$springboard bootstrap
```

Comment:
What does this do? We normally just do syncdb, migrations and collectstatic in this step. For UC stuff it's usually a resync (but the prod ones are broken iirc)